assets/src/edit-story/masks/output.js

```diff
@@ -83,7 +83,7 @@ export default function WithMask({
         ...style,
         clipPath: `url(#${maskId})`,
         // stylelint-disable-next-line property-no-vendor-prefix
-        WebkitClipPath: `url(#${maskId})`,
+        webkitClipPath: `url(#${maskId})`,
```

Comment:
Why was this changed? See #791

Reply:
Eh? 😕  I didn't change this knowingly. Odd, for some reason locally it didn't update it correctly when merging. Will change manually.

Reply:
Looks like it was changed automatically locally. Odd.

Reply:
`npm run lint:css:fix assets/src/edit-story/masks/output.js` changes this file for me locally. Every. Time. FYI.

Reply:
Good to know, thanks! Will investigate!

Reply:
Tracking in https://github.com/google/web-stories-wp/issues/845 so that I can merge this now